### ISSUE TITLE
suggestion: make loading config from env optional

### ISSUE THREAD

**Issue by schwarzkopfb** [denodrivers/postgres#418]
When I add the project as a dependency, it'll require env var access for `PGAPPNAME`, `PGDATABASE`, `PGHOST`, `PGOPTIONS`, `PGPASSWORD`, `PGPORT`, `PGUSER` regardless I prefer to use this way of providing configuration or not. I consider allowing access to all env vars via `--allow-env` as a security risk and runnig with `--allow-env=PGAPPNAME,PGDATABASE,PGHOST,PGOPTIONS,PGPASSWORD,PGPORT,PGUSER` is pretty unconvinient.

So WDYT about considering a way to opt-out from auto-config via env vars?

**Comment by canac** [comment]
I agree! I would love to be able to turn off trying to read the config from environment variables.
